packages/app-mobile/components/NoteEditor/MarkdownToolbar/Toolbar.tsx

```diff
@@ -77,7 +82,7 @@ const Toolbar = (props: ToolbarProps) => {
 		mainButtons.push(toggleOverflowButton);
 		mainButtons.push(...allButtonComponents.slice(-maxButtonsEachSide));
 	} else {
-		mainButtons.push(...allButtonComponents);
+		mainButtons.push(toggleOverflowButton);
```

Comment:
This change prevents all buttons from briefly being shown in the toolbar just after being shown again.